lib/eventum/class.mime_helper.php

```diff
@@ -177,13 +177,17 @@ public static function encodeAddress($address)
         if (self::is8bit($address)) {
             // split into name and address section
             preg_match('/(.*)<(.*)>/', $address, $matches);
+
+            $qq = preg_replace_callback(
+                '/([\x80-\xFF]|[\x21-\x2F]|[\xFC]|\[|\])/',
+                function ($m) {
+                    return '=' . strtoupper(dechex(ord(stripslashes($m[1]))));
```

Comment:
It took me a few minutes to figure out what this was doing.. as I don't use multi-byte characters.

@glensc Does this encoding need to be done manually?  Eg.  [mb_encode_mimeheader](http://php.net/manual/en/function.mb-encode-mimeheader.php)

Eg.
````
$str1 = '"Some Öne" <Some.One@example.org>';
$str2 = 'root@example.org';
$str3 = 'Root Mäe <root2@example.org>';

$encoded1 = mb_encode_mimeheader($str1, 'utf-8', "B");
echo "1: $encoded1\n";

$encoded2 = mb_encode_mimeheader($str2, 'utf-8');
echo "2: $encoded2\n";

$encoded3 = mb_encode_mimeheader($str3, 'utf-8');
echo "3: $encoded3\n";
````

Note that I didn't take the time to split the name and address portions which is already being done on L179. "B" Encoding is the default and could be omitted.